scss/_forms.scss

```diff
@@ -53,62 +53,38 @@
 
   .form-control {
     padding-right: 1.5625rem;
-    background-image: none;
-  }
-
-  &:not(.multiple)::after {
-    @include material-icon("\E876");
-    position: absolute;
-    right: .3125rem;
-    bottom: 1.8rem;
-    font-size: 1rem;
-    font-weight: $font-weight-normal;
-    vertical-align: middle;
-  }
-
-  &.multiple {
-   .input-group-input {
-    &::after {
-      @include material-icon("\E876");
-      position: absolute;
-      right: .625rem;
-      top: 50%;
-      transform: translateY(-50%);
-      font-size: 1.438rem;
-      font-weight: $font-weight-normal;
-      vertical-align: middle;
-    }
-   }
-  }
-
-  &.form-group-lg {
-    &::after {
-      bottom: 1.875rem;
-    }
+    background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' style='width:24px;height:24px' viewBox='0 0 24 24'%3E%3Cpath fill='%23f54c3e' d='M13,13H11V7H13M13,17H11V15H13M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2Z' /%3E%3C/svg%3E");
```

Comment:
What is the support for SVG embedded for background image ?

Reply:
https://caniuse.com/#feat=mdn-css_properties_content_url

Same as bootstrap, did the same exact workflow as they use